internal/verifier/sigstore/container/container.go

```diff
@@ -172,7 +172,7 @@ func getSigstoreBundles(
 ) ([]sigstoreBundle, error) {
 	imageRef := BuildImageRef(registry, owner, artifact, version)
 	// Try to build a bundle from the OCI image reference
-	bundles, err := bundleFromOCIImage(ctx, imageRef, newGithubAuthenticator(owner, auth.accessToken))
+	bundles, err := bundleFromOCIImage(ctx, imageRef, &authn.Basic{Username: owner, Password: auth.accessToken})
 	if errors.Is(err, ErrProvenanceNotFoundOrIncomplete) || errors.Is(err, ErrAuthNotProvided) {
```

Comment:
I think we can remove the check for `ErrAuthNotProvided` since it's no longer needed along with the error type 👍

Reply:
I see it's still used in `getAttestationReply` https://github.com/stacklok/minder/pull/2572/files#diff-1f0b608f02e8d9dc1bfdefad57e74550e59c5b9acd2e727369a8b177ceede25eL240

Reply:
We can replace it with a generic error type, i.e `fmt.Errorf()` as it is not compared anywhere else afterwards 👍